server/service/src/sync/synchroniser.rs

```diff
@@ -112,7 +112,7 @@ impl Synchroniser {
         let is_initialised = remote_sync_state.initial_remote_data_synced()?;
         // Initialisation request was sent and successfully processed
         let is_sync_queue_initialised = remote_sync_state.sync_queue_initalised()?;
-        // Get site id from sever
+        // Get site id from server
```

Comment:
@andreievg Just checking I should perhaps change this to _central server_ to be very clear
```suggestion
        // Get site id from central server
```

Reply:
clear as sync 🙏